test/controllers/games_controller_test.rb

```diff
@@ -1,7 +1,44 @@
-require "test_helper"
+require 'test_helper'
 
 class GamesControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  include ParsedResponse
+  include GameAsserts
+  include GamesControllerSupport
+
+  test 'should get all games on index' do
+    get_games_index
+
+    assert_response :success
+    assert_equal Game.all.size, response_data.size, 'Index games'
```

Comment:
Jose es mejor hacer variables muy semánticas y descripciones en los test más detalladas, así como asserts de respuestas, porque recuerda que esta es la documentación del código y no solo necesito un match de conteo, también necesito saber las llaves y etc porque si llego a modificar algo sin culpa digamos una llave, entonces el test del index no me indicará el error

```suggestion
    assert_response :success
    
    all_games_size = Game.all.size
    response_games_size = response_data.size

    assert_equal all_games_size, response_games_size, 'Size of all games'
```